pkg/prommetrics/prommetrics.go

```diff
@@ -109,6 +110,16 @@ var (
 		},
 		[]string{"type", "namespace"},
 	)
+
+	scaledObjectPauseTotal = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: DefaultPromMetricsNamespace,
+			Subsystem: "scaled_object",
+			Name:      "pause",
```

Comment:
I think "paused" might be better than "pause"
```suggestion
			Name:      "paused",
```

Reply:
Done